src/FuseGPUThreadLoops.cpp

```diff
@@ -1151,7 +1153,6 @@ class InjectThreadBarriers : public IRMutator {
         }
 
         return IRMutator::visit(op);
-        return op;
```

Comment:
How is it possible that none of our compilers complained about this

Reply:
Oh jeez.